### ISSUE TITLE
Nagstamon 3.8.0 missing on Flathub

### ISSUE THREAD

**Issue by Kharag** [HenriWahl/Nagstamon#789]
Hey, 
I installed Nagstamon 3.6 via Flathub (there's no package for OpenSUSE yet) and wanted to update to version 3.8.
Flatpak shows no available updates though and https://flathub.org/apps/details/de.ifw_dresden.nagstamon still refers to 3.6.0

Could you maybe update the version on Flathub?

Thanks and kind regards.

**Comment by HenriWahl** [comment]
Correct... takes some time to update the yaml files at https://github.com/flathub/de.ifw_dresden.nagstamon... any help is welcome. I already updated locally

- https://github.com/flathub/de.ifw_dresden.nagstamon/blob/master/de.ifw_dresden.nagstamon.appdata.xml
- https://github.com/flathub/de.ifw_dresden.nagstamon/blob/master/de.ifw_dresden.nagstamon.yml
- https://github.com/flathub/de.ifw_dresden.nagstamon/blob/master/modules/python3-beautifulsoup4.yml

and plan to update the others too. If you want to speed up this process just do a fork and help to update the remaining files :-)

**Comment by HenriWahl** [comment]
I am not able to get it running with newer dependencies. Maybe someone else has more flatpak skills?

**Comment by csarn** [comment]
Any news? flatpak complains about some dependencies of nagstamon being end-of-life...

**Comment by HenriWahl** [comment]
Sad to say no. I failed to create an updated flatpak and found no time to try again. Maybe someone out there will succeed?

**Comment by HenriWahl** [comment]
Now available as 3.18.2.
